### DISCUSSION TITLE
What's the status of ImageGlass-Photos?

### DISCUSSION THREAD

**Discussion by Keith94** [d2phap/ImageGlass#439] [General] [answered]
https://github.com/d2phap/ImageGlass-Photos

Will this new version be migrated here, or will development switch to the new repo?

**Comment by d2phap** [comment, marked as answer]
This is not a replacement of the .NET version.
It's just an experimental version using web technology (JavaScript, HTML, CSS).

With web technology, the UI can be very flexible with fancy effects that I cannot achieve in the .NET version. The speed is faster because it takes advantage of GPU power. And it supports multiple platforms: Windows OS, Mac OS, Linux...

However, there are some disadvantages: 
- Big file size (> 100 MB)
- High DPI scaling issue (non-scaled image formats like PNG, JPG,... are blurry)
- Less image formats supported
- I need to start developing from 0
- ...

If the Web version can come true, it will be like another option beside ImageGlass.

**Comment by smaragdus** [comment]
@d2phap 

JavaScript/Electron programs have other disadvantages- excessive memory usage, not native look and feel. I really hope that the .NET version of ImageGlass will not be abandoned in favour of the Electron version (I avoid Electron programs like plague).

**Comment by d2phap** [comment]
Thanks @smaragdus for the additional info.
As I mentioned, I have no plan for Electron version and it is not a replacement of .NET version.

**Comment by Keith94** [comment]
Ok, thanks. That answers my question 👍
